arches_lingo/src/arches_lingo/components/login/LoginLinks.vue

```diff
@@ -13,11 +15,15 @@ const { $gettext } = useGettext();
             as="a"
             :label="$gettext('Register')"
             :href="arches.urls.signup"
+            :severity="SECONDARY"
+            :outlined="true"
         />
         <Button
```

Comment:
nit, and not necessarily part of this PR, but the text in the button is underlined and arguably shouldn't be.

Reply:
My thought was that it's a better practice for links to be underlined (at least that's what I'm sure WCAG says), and this button is really a link (`as="a"`)

Reply:
oh fair. Not to be handled in this ticket but I could see abandoning the button entirely then and just having links. But IMO wait for that order to come from someone with a stronger opinion 👍